contents/blog/rudderstack-integration.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2020-11-19
+title: PostHog is Now Available on RudderStack!
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+<br>
+
+If you didn't already know, we're [big fans of open source](/blog/open-source-eating-saas).
+
+As such, we're happy to announce an integration with yet another open source platform: [RudderStack](rudderstack.com).
+
+RudderStack is an open-source, warehouse-first, customer data platform for developers. It allows you to collect and deliver customer event data to a variety of destinations such as data warehouses and analytics platforms.
+
+![RudderStack Pipeline](../images/blog/rs/rudderstack-pipeline.png)
+
+The idea is that you can collect data from various sources and send it to one of their 70+ destinations with a single integration, making it easy to aggregate data from all services you use for data collection, as well as get the aggregated data into a data warehouse or sent to all the services where you need it to be. 
+
+For example, **with a single SDK**, you can collect usage data from both your apps and websites and then send them over to:
```

Comment:
I presume it's not possible to use PostHog's data collection -> PostHog -> Rudderstack -> Other stuff?

(Just because if we promote using Rudderstack around us, we're demoting ourselves here for users that currently use us for data collection - they're probably using the Segment open source libraries anyway..!)

Reply:
We're only a destination on RudderStack, not a source i.e. data comes in not out of PostHog